### ISSUE TITLE
Error loading the model depth and Hed (Only them)

### ISSUE THREAD

**Issue by fabiobtex** [Mikubill/sd-webui-controlnet#355]
All other models works normally, all preprocessors works (including Depth and Hed), i already reinstalled everything, already deleted venv and already download the pth again. What could be?  

Loading model: control_sd15_depth [fef5e48e]
Error running process: D:\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py
Traceback (most recent call last):
  File "D:\stable-diffusion-webui\modules\scripts.py", line 386, in process
    script.process(p, *script_args)
  File "D:\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 571, in process
    else self.build_control_model(p, unet, model, lowvram)
  File "D:\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 433, in build_control_model
    state_dict = load_state_dict(model_path)
  File "D:\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\utils.py", line 11, in load_state_dict
    state_dict = get_state_dict(torch.load(
  File "D:\stable-diffusion-webui\extensions\sd_dreambooth_extension\reallysafe.py", line 117, in load
    return load_with_extra(filename, *args, **kwargs)
  File "D:\stable-diffusion-webui\extensions\sd_dreambooth_extension\reallysafe.py", line 164, in load_with_extra
    return unsafe_torch_load(filename, *args, **kwargs)
  File "D:\stable-diffusion-webui\modules\safe.py", line 106, in load
    return load_with_extra(filename, extra_handler=global_extra_handler, *args, **kwargs)
  File "D:\stable-diffusion-webui\modules\safe.py", line 151, in load_with_extra
    return unsafe_torch_load(filename, *args, **kwargs)
  File "D:\stable-diffusion-webui\venv\lib\site-packages\torch\serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "D:\stable-diffusion-webui\venv\lib\site-packages\torch\serialization.py", line 1131, in _load
    result = unpickler.load()
  File "D:\stable-diffusion-webui\venv\lib\site-packages\torch\serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "D:\stable-diffusion-webui\venv\lib\site-packages\torch\serialization.py", line 1079, in load_tensor
    storage = zip_file.get_storage_from_record(name, numel, torch.UntypedStorage).storage().untyped()
PermissionError: [Errno 13] Permission denied

**Comment by relocator** [comment]
I was getting a similar error, but updating Automatic1111 fixed my problems. I'm assuming you tried that, but just in case!
"

**Comment by fabiobtex** [comment]
Mine was updated both automatic1111 and the extension.

My problem started when I moved the automatic1111 folder to a new SDD, a heard it's a ok procedure, i fixed the paths in settings on the controlnet part, everything worked fine except by those two models, even deleting the whole controlnet folder didn't worked, so i deleted venv and repositores folders a ran again and still no solved.

Maybe there is a need to fix more paths that is not in the automatic1111 settings.

Coincidence or not my new ssd stopped working, today I did the whole installation again with 100% new files (I lost everything I had) and now it's working again.

Unfortunately I can't try to reproduce the problem as I don't have another SSD anymore but i hope that helps to close this issue.
